study.md

```diff
@@ -27,17 +27,20 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+Dom is not a flat markup language, needs to be always in the pair with html if we want to see the content not only in the console but also in the browser window.
+
+Dom is not view source.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes, the tags with notes in the browser inspect element tool are part of the tree structure as called the document object model.
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.